introductory/src/App.jsx

```diff
@@ -1,9 +1,11 @@
 
 
-function App() {
+export default function App() {
     return (
```

Comment:
There is no need to export this function because exporting functions that are not needed externally can lead to unnecessary dependencies

Reply:
Actually App function is being rendered in main.jsx, so I don't think  there is a way of rendering it in main without exporting it. Probably you can enlighten me.